src/picongpu/include/simulation_defines/param/speciesAttributes.param

```diff
@@ -55,13 +55,13 @@ value_identifier(float_X, weighting, 0.0);
 /** use this particle for radiation diagnostics */
 value_identifier(bool, radiationFlag, false);
 
-/** charge state of the particle
+/** number of electrons bound to the atom / ion
  *
  * value type is float_X to avoid casts during the runtime
- *
+ * - float is also reasonable because effective charge numbers are possible
  * - only reasonable for ion species if ionization is enabled 
- * - \todo maybe calculate from new attribute boundElectrons */
-value_identifier(float_X,chargeState,float_X(1.0));
+ */
+value_identifier(float_X,boundElectrons,float_X(1.0));
```

Comment:
Probably `1.0` is not the best "general" default value.
Maybe "fully ionized" aka `float_X(0.0)` is a better choice for now (or later on and alternatively, the proton-number for "neutral").